db/migrate/20160818210955_add_comment_to_secrets.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddCommentToSecrets < ActiveRecord::Migration
+  def change
+    add_column :secrets, :comment, :string
```

Comment:
I'd hope so ... if not we can expand easily

On Thu, Aug 18, 2016 at 4:11 PM, Jon Moter <notifications@github.com> wrote:

> In db/migrate/20160818210955_add_comment_to_secrets.rb
> <https://github.com/zendesk/samson/pull/1228#discussion_r75405581>:
>
> > @@ -0,0 +1,6 @@
> > +# frozen_string_literal: true
> > +class AddCommentToSecrets < ActiveRecord::Migration
> > +  def change
> > +    add_column :secrets, :comment, :string
>
> Think the default 256-character limit is sufficient for a comment?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1228/files/7f8d73ef0733cc745d85f3274a9eaa75e7761cb6#r75405581>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ3MKDXuCQrxlQ0AIkjdhBNemBMFvks5qhOaVgaJpZM4Jn-tr>
> .
>